readthedocs/builds/managers.py

```diff
@@ -62,7 +61,6 @@ def create_latest(self, **kwargs):
             'verbose_name': LATEST_VERBOSE_NAME,
             'machine': True,
             'active': True,
-            'identifier': LATEST,
```

Comment:
Curious about the reason for this change. Should we be doing the same thing for `stable` above? 🤔

Reply:
Probably. I'm not 100% sure the workflow about "how `stable` is defined" --I should have to investigate more about that. However, `stable` shouldn't be defined _before_ the first VCS sync where we grab all the branches/tags of the repository. That said, in theory, it makes sense to be `None/NULL` initially.

I don't want to open that door in this PR, tho 😄 , but I'm happy to continue with this investigation and idea in a issue since the change makes sense to me.

Reply:
Makes sense. Perhaps add a `TODO?